src/Management/src/Endpoint/ManagementHostBuilderExtensions.cs

```diff
@@ -329,6 +330,23 @@ public static IHostBuilder AddTraceActuator(this IHostBuilder hostBuilder)
         return AddTraceActuator(hostBuilder, MediaTypeVersion.V2);
     }
 
+    /// <summary>
+    /// Adds the Services actuator to the application.
+    /// </summary>
+    /// <param name="hostBuilder">
+    /// Your HostBuilder.
+    /// </param>
```

Comment:
Documentation should be improved, as suggested at https://github.com/SteeltoeOSS/Steeltoe/pull/1129#discussion_r1376307264. Postponing until #1223 has been merged, as it shuffles things around.